ansible.cfg

```diff
@@ -1,5 +1,4 @@
 [defaults]
-roles_path = ~/.ansible/roles:/usr/share/ansible/roles:/etc/ansible/roles:roles
```

Comment:
This will break ad-hocs runs though.

Reply:
should we support ad-hoc runs?

Reply:
I run ad-hoc runs as I am testing things somtimes. Against a random container, VM or EC2 instance.

Not sure if `molecule` on the command line would be broken or not. But I use that all the time.

We had a discussion about this like a year ago with @mdellweg